tests/data/MerchFactory.ts

```diff
@@ -110,6 +110,7 @@ export class MerchFactory {
       orderLimit: FactoryUtils.getRandomNumber(1, 5),
       status: OrderPickupEventStatus.ACTIVE,
       orders: [],
+      linkedEvent: null,
```

Comment:
Should I import the EventFactory and generate an actual dummy event to put in here? Are there any other test cases that could be beneficial for this feature?

Reply:
I don't actually think so. I actually don't think it is clean to attach a dummy event everytime we mock a pickupEvent as sometimes pickupEvent doesn't have an actual linkedEvent attached.

I don't believe it is necessary to implement a solution for this now, but for future reference, if there is a new feature that actually depends on linkedEvent, we should come up with a nice way (such as taking an optional boolean flag variable to attach event) to attach dummy linkedEvent to the mocked pickupEvent.